apps/console/src/styles/app.css

```
@@ -849,3 +849,228 @@ input:focus, textarea:focus, select:focus {
+.drawer-header {
+  flex: none;
+  padding: 16px 18px 14px;
+  border-bottom: 1px solid var(--app-border);
+  display: flex; flex-direction: column; gap: 8px;
+}
+.drawer-header-top {
+  display: flex; align-items: flex-start; justify-content: space-between; gap: 12px;
+}
+.drawer-title-h2 {
+  margin: 2px 0 0;
+  font-family: 'Lora', Georgia, serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Lora" (font-family-name-quotes)

(font-family-name-quotes)